src/Tizen.NUI/src/internal/FriendAssembly.cs

```diff
@@ -18,18 +18,43 @@
+[assembly: InternalsVisibleTo("Tizen.Multimedia, " + PublicKey.TizenFX)]
+[assembly: InternalsVisibleTo("Tizen.NUI.Wearable, " + PublicKey.TizenFX)]
+[assembly: InternalsVisibleTo("Tizen.NUI.Components, " + PublicKey.TizenFX)]
 
-//This is for Layout development and test. snk file will be shared internally. This will be removed at 2018-09-30.
-[assembly: InternalsVisibleTo("Tizen.TV.NUI.Example, PublicKey=0024000004800000940000000602000000240000525341310004000001000100ed445c2a988d35b99e5766f42eef33d89d1f67225db051f011abcfce47a4788875c39750a2e5695f1ec117f41d96610419811762669f98131db30a6e64c6bb8fde3731f373f6cda4c0087d121f5108559f216dc015807dc46ec5a4d1b63f5deff64c01754a0db0dc849bc300672572cbd2697432ab6c193ebf9fade6bf0f2aad")]
-[assembly: InternalsVisibleTo("NuiSample, PublicKey=0024000004800000940000000602000000240000525341310004000001000100a5bcf6fb6095141fa8146ec74d5b0be8b09a063f8f1f32604b10df6e498788182840bd15c93ba1a5c0b9aeee8fe2be42ea9c0b5fe32e1d6d5602b2a4fa813114fd58c4b2910d93e5181b90925458b4b8b003c30d97830a17ad90a92f34f60a11f67a78757e6b2035e799be5ab69a41ede17cad2e2b7967fbf49b685452496be0")]
+[assembly: InternalsVisibleTo("Tizen.TV.NUI, " + PublicKey.TizenTV)]
+[assembly: InternalsVisibleTo("Tizen.TV.NUI.Component, " + PublicKey.TizenTV)]
 
-[assembly: InternalsVisibleTo("Tizen.TV.FLUX, PublicKey=0024000004800000940000000602000000240000525341310004000001000100ed445c2a988d35b99e5766f42eef33d89d1f67225db051f011abcfce47a4788875c39750a2e5695f1ec117f41d96610419811762669f98131db30a6e64c6bb8fde3731f373f6cda4c0087d121f5108559f216dc015807dc46ec5a4d1b63f5deff64c01754a0db0dc849bc300672572cbd2697432ab6c193ebf9fade6bf0f2aad")]	
-[assembly: InternalsVisibleTo("Tizen.TV.FLUX.Component, PublicKey=0024000004800000940000000602000000240000525341310004000001000100ed445c2a988d35b99e5766f42eef33d89d1f67225db051f011abcfce47a4788875c39750a2e5695f1ec117f41d96610419811762669f98131db30a6e64c6bb8fde3731f373f6cda4c0087d121f5108559f216dc015807dc46ec5a4d1b63f5deff64c01754a0db0dc849bc300672572cbd2697432ab6c193ebf9fade6bf0f2aad")]
-[assembly: InternalsVisibleTo("Tizen.TV.FLUX.Example, PublicKey=0024000004800000940000000602000000240000525341310004000001000100ed445c2a988d35b99e5766f42eef33d89d1f67225db051f011abcfce47a4788875c39750a2e5695f1ec117f41d96610419811762669f98131db30a6e64c6bb8fde3731f373f6cda4c0087d121f5108559f216dc015807dc46ec5a4d1b63f5deff64c01754a0db0dc849bc300672572cbd2697432ab6c193ebf9fade6bf0f2aad")]
+[assembly: InternalsVisibleTo("Tizen.TV.FLUX, " + PublicKey.TizenTV)]
+[assembly: InternalsVisibleTo("Tizen.TV.FLUX.Component, " + PublicKey.TizenTV)]
+[assembly: InternalsVisibleTo("Tizen.TV.FLUX.Example, " + PublicKey.TizenTV)]
+[assembly: InternalsVisibleTo("Tizen.TV.NUI.Example, " + PublicKey.TizenTV)]
+
+[assembly: InternalsVisibleTo("Tizen.TV.NUI.Example, " + PublicKey.Sample)]
```

Comment:
this is same app name(Tizen.TV.NUI.Example) as above. maybe this is NuiSample which is for Ubuntu nui sample project. could you check ?

Reply:
Fixed